config/routes.rb

```diff
@@ -33,6 +33,7 @@ def matches?(request)
         # i.e the request 'authn-oidc/:service_id/:account/authenticate' can be interpreted as
         # ':authenticator/:account/:id/authenticate'
         post '/authn-oidc(/:service_id)/:account/authenticate' => 'authenticate#authenticate_oidc'
+        post '/authn-jwt(/:service_id)/:account(/:identity)/authenticate' => 'authenticate#authenticate_jwt'
```

Comment:
```suggestion
        post '/authn-jwt(/:service_id)/:account(/:id)/authenticate' => 'authenticate#authenticate_jwt'
```

change to `id` to match the request below